doc/how_to/interactivity/bind_generators.md

```diff
@@ -11,7 +11,11 @@ import panel as pn
 pn.extension()
 ```
 
-Let us say we have some action that is triggered by a widget, such as a button, and while we are computing the results we want to provide feedback to the user. Using imperative programming this involves writing callbacks that update the current state of our components. This is complex and really we prefer to write reactive components. This is where reactive generators come in.
+Let us say we have some action that is triggered by a widget, such as a button, and while we are computing the results we want to provide feedback to the user. Using imperative programming this involves writing callbacks that update the current state of our components. This is complex and really we prefer to write reactive components. This is where *generator functions* come in.
+
+:::{important}
```

Comment:
We experienced I was not able to create a generator function. I've added the information that would have helped me avoid that.

Reply:
Yup this is great.